node/bft/src/gateway.rs

```diff
@@ -970,6 +985,20 @@ impl<N: Network> Gateway<N> {
         }
     }
 
+    #[cfg(feature = "telemetry")]
+    // Logs the validator participation scores.
+    fn log_participation_scores(&self) {
+        if let Ok(current_committee) = self.ledger.current_committee() {
+            // Retrieve the participation scores.
+            let participation_scores = self.validator_telemetry().get_participation_scores(&current_committee);
+            // Log the participation scores.
+            debug!("Participation Scores (in the last {} rounds):", self.storage.max_gc_rounds());
+            for (address, score) in participation_scores {
```

Comment:
nit: if we expect to not always be running with debug logs, it would be best to make the execution of this loop conditional based on the log level